apis/externalsecrets/v1beta1/externalsecret_types.go

```diff
@@ -468,10 +470,14 @@ type ExternalSecret struct {
 }
 
 const (
-	// AnnotationDataHash is used to ensure consistency.
+	// AnnotationDataHash all secrets managed by an ExternalSecret have this annotation with the hash of their data.
 	AnnotationDataHash = "reconcile.external-secrets.io/data-hash"
-	// LabelOwner points to the owning ExternalSecret resource
-	//  and is used to manage the lifecycle of a Secret
+
+	// LabelManaged all secrets managed by an ExternalSecret will have this label equal to "true".
+	LabelManaged      = "reconcile.external-secrets.io/managed"
```

Comment:
The issue is that not all resources that we are "managing" (and need to cache) have the `LabelOwner` label, also because multiple ExternalSecrets can target the same Secret (with CreationPolicy=Merge) it would be impossible to safely set on all target secrets.